### ISSUE TITLE
HTTP console interface error: The element type "meta" must be terminated by the matching end-tag "</meta>".

### ISSUE THREAD

**Issue by esabol** [UniversalMediaServer/UniversalMediaServer#2912]
UMS 10.19 (but I've seen this with all the versions I've tried going back to 10.13, for the record) TRACE-level debug log:
```
INFO  2022-03-31 18:04:14.812 [HTTPv2 Request Worker 1] Media renderer was not recognized. Possible identifying HTTP headers:
User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_#_#) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/99.0.4844.83 Safari/537.36
Upgrade-Insecure-Requests: 1
DNT: 1
Referer: http://192.168.1.###:500#/console/stop
TRACE 2022-03-31 18:04:14.812 [HTTPv2 Request Worker 1] Received a request from PlayStation 4 (192.168.1.###:#####):

GET /console/home HTTP/1.1

HEADER:
  Host: 192.168.1.###:500#
  Connection: keep-alive
  Upgrade-Insecure-Requests: 1
  DNT: 1
  User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_#_#) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/99.0.4844.83 Safari/537.36
  Accept: text/html,application/xhtml+xml,application/xml;q=0.9,image/avif,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.9
  Referer: http://192.168.1.###:500#/console/stop
  Accept-Encoding: gzip, deflate
  Accept-Language: en-US,en;q=0.9

Renderer UUID=null
INFO  2022-03-31 18:04:14.815 [HTTPv2 Request Worker 1] [Fatal Error] :1:87: The element type "meta" must be terminated by the matching end-tag "</meta>".
TRACE 2022-03-31 18:04:14.815 [HTTPv2 Request Worker 1] Response sent to PlayStation 4:

HEADER:
  HTTP/1.1 200 OK
  Content-Type: text/html
  Server: Mac_OS_X-x86_64-10.#.#, UPnP/1.0 DLNADOC/1.50, UMS/10.19.0
  Content-Length: 308

CONTENT:
  Content isn't valid XML, using text formatting: The element type "meta" must be terminated by the matching end-tag "</meta>".
    <html><head><meta charset="utf-8"><title>Universal Media Server HTML Console</title></head><body><p style="margin: 0 auto; text-align: center;"><img src='/images/logo.png'><br>Universal Media Server HTML console<br><br>Menu:<br><a href="home">Home</a><br><a href="scan">Scan folders</a><br></p></body></html>
```

**Comment by SubJunk** [comment]
@esabol thanks for reporting this, will fix it in the next version

**Comment by SurfaceS** [comment]
It is not an error.
HTML answer is not XHTML or XML.
Closing with "</meta>" will break on some browser as it doesn't comply with HTML.
This error is due to the trace (and only trace) logger that try to parse the answer as XML (for soap) to nicely show the answer.

You will have the same problem with <img>, <br> etc as HTML is not XML.

**Comment by SubJunk** [comment]
My mistake, thanks

**Comment by esabol** [comment]
If it's not an error, then words "fatal error" should not appear in the logs! Maybe don't validate your HTML as XML? Either way, that's a bug in my book, but, hey, I've just been a professional programmer for 27 years, so what do I know.

**Comment by SubJunk** [comment]
Actually yeah I think you are right @esabol 
I have started a branch for this. One problem was we were not declaring a doctype with the HTML. I will do some more testing of it

**Comment by SubJunk** [comment]
After looking at it some more I think we need to check the content-type header before attempting to parse it as XML. The only reason we are parsing it is for logging anyway so the stakes are very low but it will be more tidy this way

**Comment by SubJunk** [comment]
This should be resolved now in #2928 
There was no bug related to it since it only affected our logging, but it was nice to tidy it up so code that was in 6 files is now in 1 file, and it will only prettify the XML if the right content-type header exists

**Comment by SubJunk** [comment]
this is fixed in the upcoming v11 release
